### DISCUSSION TITLE
What do you think if we can use another pocketbase for storing ?

### DISCUSSION THREAD

**Discussion by hungcrush** [pocketbase/pocketbase#2501] [Q&A] [answered]
I have unlimited hosting, which allows me to access SSH and start the Pocketbase. However, the hosting's location is far from where I am located. Therefore, my idea is to use that hosting as my storage service and have another Pocketbase utilize it. 

What do you think about this?

**Comment by ganigeorgiev** [comment, marked as answer]
If you just want to use the other instance for storing uploaded files it may be better to install something like [minio](https://github.com/minio/minio) and connect PocketBase to it with its S3 interface.

Alternatively, if you want to have 2 synced PocketBase instances, you can try [marmot](https://github.com/maxpert/marmot/) or [litefs](https://github.com/superfly/litefs), but keep in mind that the hooks and realtime events are not db synced.

**Reply by hungcrush** [reply]
Thank you so much. This is first time I hear about [minio](https://github.com/minio/minio).. Thank you....
